File: src/sumfp/sumfp-ignore-setuptime.js
var run =function(n){
    var loop =function (i, sum){
        if (i<0.0)
            return sum;
        return loop(i-1, i+ sum);
    };
    return loop(n, 0.0);
}

for(var i=0; i< 10000;i++){
    run(8000);
}

File: src/sumfp/sumfp-ignore-setuptime.py
import sys

sys.setrecursionlimit(1000000)

def sum(n):
    def sum_helper(n, acc):
        if n < 0:
            return acc
        else:
            return sum_helper(n-1, acc+n)
    return sum_helper(n, 0.)

for i in range(0,10000)
    sum(8000)